pandas/plotting/_core.py

```diff
@@ -2852,22 +2852,47 @@ def pie(self, y=None, **kwds):
 
     def scatter(self, x, y, s=None, c=None, **kwds):
         """
-        A scatter plot with point size `s` and color `c`.
+        Create a scatter plot with varying marker point size and color.
 
-        The coordinates of each point `x,y` are defined by two dataframe
-        columns and filled circles are used to represent each point.
+        The coordinates of each point are defined by two dataframe columns and
+        filled circles are used to represent each point. This kind of plot is
+        useful to see complex correlations between two variables. Points could
+        be for instance natural 2D coordinates like longitude and latitude in
+        a map or, in general, any pair of metrics that can be plotted against
+        each other.
 
         Parameters
         ----------
-        x : column name or column position
-            Horizontal coordinates of each point.
-        y : column name or column position
-            Vertical coordinates of each point.
-        s : scalar or array_like, optional
-            Size of each point.
-        c : label, column name or column position, optional
-            Color of each point.
-        kwds : optional
+        x : int, str
+            The column name or column position to be used as horizontal
+            coordinates for each point.
+        y : int, str
+            The column name or column position to be used as vertical
+            coordinates for each point.
+        s : scalar, array_like, optional
+            The size of each point. Possible values are:
+
+            - A single scalar so all points have the same size.
+
+            - A sequence of scalars, which will be used for each point's size
+            recursively. For intance [2,14] all points will be size 2 or 14,
+            alternatively.
+
+        c : str, int, array_like, optional
+            The color of each point. Possible values are:
+
+            - A single color string referred to by name, RGB or RGBA code,
```

Comment:
Have you verified if these bullet points render nicely in the final HTML? I'm not good at restructured text so I tend to be wary about these things. :)

Reply:
You need the blank lines to properly render the bullet points ;)
<img width="714" alt="screen shot 2018-03-14 at 09 59 01" src="https://user-images.githubusercontent.com/10695317/37392431-70bfe218-276e-11e8-9e4e-395bd767b788.png">